lib/fs.js

```diff
@@ -2033,7 +2033,7 @@ function ReadStream(path, options) {
     // (That is a semver-major change).
     this.end = Infinity;
   } else if (!Number.isInteger(this.end)) {
```

Comment:
As we decided to support `{end:Infinity}`, change `else if (!Number.isInteger(this.end))` to `else if (!(Number.isInteger(this.end)||this.end===Infinity))`.

Reply:
Also change line 2019. (Comments are not allowed there.)